src/gluonnlp/attention_cell.py

```diff
@@ -311,7 +311,7 @@ def masked_logsoftmax(att_score, mask, dtype=np.float32, axis: int = -1):
     if mask is not None:
         # Fill in the masked scores with a very small value
         neg = -1e18
-        if np.dtype(dtype) == np.float16:
+        if _np.dtype(dtype) == np.float16:
```

Comment:
Do we have to change this line?

Reply:
Yes. Otherwise there will be `ValueError: invalid __array_struct__`

Reply:
@szha @leezu Should be one issue of MXNet numpy.

Reply:
Would you open an issue with a minimal example about this in the MXNet repo @barry-jin? If you like, you may also open a PR to fix it later. For now, thanks for finding the workaround for gluonnlp.

Reply:
Ok, I will.